record list item: add compact stats component

jennur opened the pull request:
Closes https://github.com/inveniosoftware/invenio-app-rdm/issues/2174

Adds new `<CompactStats />` component to all record list items (My dashboard - mobile & computer comp., search results).


## Screenshots 
![Screenshot 2023-03-31 at 15 23 38](https://user-images.githubusercontent.com/21052053/229136716-cca87867-a728-4536-aa5e-203bbdfba0e0.png)
![Screenshot 2023-03-31 at 15 23 49](https://user-images.githubusercontent.com/21052053/229136719-b09a4896-316e-4fd3-96fa-870466612e10.png)
![Screenshot 2023-03-31 at 15 23 53](https://user-images.githubusercontent.com/21052053/229136724-cba19207-fafa-4ef3-8ab6-9f0366a97528.png)
![Screenshot 2023-03-31 at 15 23 55](https://user-images.githubusercontent.com/21052053/229136725-2368149b-59ac-496e-8b72-10f8963285f5.png)
![Screenshot 2023-03-31 at 15 24 08](https://user-images.githubusercontent.com/21052053/229136860-5aa4d680-df74-4b6c-b7bb-7b2ada473540.png)

kpsherva commented on invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/components/CompactStats.js:
```diff
@@ -0,0 +1,52 @@
+import { Popup, Label, Icon } from "semantic-ui-react";
+
+export const CompactStats = ({ uniqueViews, uniqueDownloads }) => {
+  console.log("Unique views:", uniqueViews);
```
👀

jennur replied:
woopsie! Thanks! Will fix

kpsherva commented on invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/MobileUploadsItem.js:
```diff
@@ -38,7 +40,8 @@ export const MobileUploadsItem = ({
     <Icon name="upload" className="negative mr-10" />
   );
-
+  const uniqueViews = _get(result, "stats.all_versions.unique_views", -1);
```
why -1 not 0 ? if -1 is displayed then it will be confusing for the users

jennur replied:
True! Didn't realize. Thanks!!

kpsherva commented:
LGTM! added small comments

jennur commented:
## Updates
- Removed `console.log`
- Replaced all occurences of
```
const uniqueViews = _get(result, "stats.all_versions.unique_views", -1);
const uniqueDownloads = _get(result, "stats.all_versions.unique_downloads", -1);
```
with
```
const uniqueViews = _get(result, "stats.all_versions.unique_views", 0);
const uniqueDownloads = _get(result, "stats.all_versions.unique_downloads", 0);
```
